frontend/src/components/custom/MatchingOptions/MatchingButton.tsx

```diff
@@ -25,16 +28,28 @@ const MatchingButton: React.FC<MatchingButtonProps> = ({
         "ws://localhost:5003/matching";
       const token = sessionStorage.getItem("authToken");
       const uid = sessionStorage.getItem("uid");
+      const otherUid = sessionStorage.getItem("otherUid");
```

Comment:
This field was not set anywhere in the frontend so it would not have a value.

Ideally, each sessionStorage should only have knowledge about the current user's information and should not store information about the others.